jest.config.js

```diff
@@ -17,5 +18,9 @@ module.exports = {
   ],
   "transform": {
     "^.+\\.ts$": "ts-jest",
+    "^.+\\.js$": "babel-jest",
   },
+  "transformIgnorePatterns": [
+    "/test/cli/fixtures/bad",
```

Comment:
Jest was trying to parse this file using Babel which was failing, so we ignore it here